pages/sentinel_cherrypick.vue

```diff
@@ -131,14 +131,14 @@ export default {
       this.handleGetPlatesResponse(resp)
     },
     handleGetPlatesResponse(resp) {
-      if (resp.length === 0) {
+      if (!resp.success) {
         this.pickListResponse = {
           alertMessage: 'Could not retrieve plates from LabWhere',
           variant: 'danger'
         }
         this.showDismissibleAlert = true
       } else {
-        this.items = resp.map((barcode) => ({
+        this.items = resp.plateBarcodes.map((barcode) => ({
```

Comment:
Is this change correct? Can't see where `plateBarcodes` is getting added as a key in the response?

Reply:
Plate barcodes is added as a key in labwhere.js

Reply:
Ah, I didn't realise `return { success: true, plateBarcodes }` added it as a key!